android/src/main/java/com/thebluealliance/androidclient/datafeed/status/TBAStatusController.java

```diff
@@ -32,30 +40,50 @@
 
     public static final String STATUS_PREF_KEY = "tba_status";
 
-    /* 15 minutes in nanoseconds */
-    private static final double UPDATE_TIMEOUT_NS = 9e+11;
+    /**
+     * We use different timeouts for logged in users and not logged in users. Logged-in users will
+     * receive push notifications when the status changes, so it's not as important to poll
+     * frequently. For logged-out users, however, we want to poll more often because they won't
+     * get push notifications.
+     */
+    // 15 minutes
+    private static final double UPDATE_TIMEOUT_LOGGED_IN_NS = 9e+11;
+    // 1 minute
+    private static final double UPDATE_TIMEOUT_LOGGED_OUT_NS = 6e+10;
+
+    // 3 hours
+    private static final double DIALOG_TIMEOUT = 1.08e+13;
 
     private final SharedPreferences mPrefs;
     private final Gson mGson;
     private final Tracker mAnalyticsTracker;
+    private final Context mContext;
 
     private long mLastUpdateTime;
     private long mLastDialogTime;
 
+    private boolean mUserIsLoggedIn;
+    @Inject Context context;
```

Comment:
@nwalters512 you can remove this one too. You don't need the annotated variable if you're using constructor injection, see [docs](http://square.github.io/dagger/)